server/rpm/pbench-server.spec

```diff
@@ -110,31 +104,33 @@ rm -rf Makefile %{static}/Makefile \
    lib/%{pyversion}/site-packages/* \
    lib/pbench/common/__pycache__ \
    lib/pbench/cli/__pycache__ \
-   lib/pbench/cli/getconf.py \
    lib/pbench/cli/agent \
    lib/pbench/server/__pycache__
 
 %post
 
 chown -R pbench.pbench /%{installdir}
 
+cd /%{installdir}
+
 # on RHEL7 we use scl to get python3
 %if 0%{?rhel} == 7
 # install python3-elasticsearch
```

Comment:
I guess this comment really wasn't accurate before since you listed boto3 and ujson as well as just elasticsearch1; it's even less accurate now that you're referencing `requirements.txt`. Maybe just "install Python dependencies"?

Reply:
Yes, I'll go through and scrub them.